### ISSUE TITLE
Multiple crashes in my modpack when i load a world

### ISSUE THREAD

**Issue by XvaleX10** [fonnymunkey/FermiumBooter#1]
[crash-2024-02-14_18.26.19-client.txt](https://github.com/fonnymunkey/FermiumBooter/files/14284263/crash-2024-02-14_18.26.19-client.txt)
[crash-2024-02-14_18.26.20-client.txt](https://github.com/fonnymunkey/FermiumBooter/files/14284264/crash-2024-02-14_18.26.20-client.txt)
[crash-2024-02-14_18.26.25-client.txt](https://github.com/fonnymunkey/FermiumBooter/files/14284265/crash-2024-02-14_18.26.25-client.txt)
[crash-2024-02-14_18.26.14-client.txt](https://github.com/fonnymunkey/FermiumBooter/files/14284266/crash-2024-02-14_18.26.14-client.txt)
[crash-2024-02-14_18.26.17-client.txt](https://github.com/fonnymunkey/FermiumBooter/files/14284267/crash-2024-02-14_18.26.17-client.txt)
[crash-2024-02-14_18.26.18-client.txt](https://github.com/fonnymunkey/FermiumBooter/files/14284268/crash-2024-02-14_18.26.18-client.txt)
This occur when i load a world with a mod that uses FermiumBooter like RLmixins or FermiumASM (In my case i use both)
When i load only FermiumBooter without any of these two mods it doesn't happen

**Comment by fonnymunkey** [comment]
I attempted to copy your mod environment however I do not crash on loading a world.

Does this happen in any world you create or only specific existing ones?
Can you upload a CF export file of your pack to somewhere like google drive including your configs?

I'm not sure that the crash is being caused by FermiumBooter nor the dependant mods, possibly some sort of config/loading conflict between multiple.

Your crash reports suggest that various LittleTiles blocks such as cables are being created clientside as falling blocks, which I'm not sure is supposed to even be possible, and isn't something my mods touch.

**Comment by XvaleX10** [comment]
> I attempted to copy your mod environment however I do not crash on loading a world.
> 
> Does this happen in any world you create or only specific existing ones? Can you upload a CF export file of your pack to somewhere like google drive including your configs?
> 
> I'm not sure that the crash is being caused by FermiumBooter nor the dependant mods, possibly some sort of config/loading conflict between multiple.
> 
> Your crash reports suggest that various LittleTiles blocks such as cables are being created clientside as falling blocks, which I'm not sure is supposed to even be possible, and isn't something my mods touch.

It's not really a crash because it doesn't exit the game but a kind of error that occurs while I explore a world loading new chunks, every time I check the modpack files I find the crash reports full of these errors
However, I did some testing and discovered that it only occurs when the FermiumASM mod is installed (It also happens with CensoredASM)
Sorry if I reported the problem here because I thought it was FermiumBooter's fault, now that I've done more tests I understand that it has nothing to do with it
Without FermiumASM 
[latest.log](https://github.com/fonnymunkey/FermiumBooter/files/14295430/latest.log)
With FermiumASM in the same world loading new chunks
[latest.log](https://github.com/fonnymunkey/FermiumBooter/files/14295449/latest.log)
This is the modpack files
https://drive.google.com/file/d/16jijNSBBfwfW2rnRzBgotmU-hJOsoULM/view?usp=drive_link

**Comment by fonnymunkey** [comment]
It happening with FermiumASM/CensoredASM and being unrelated to RLMixins makes a lot more sense, as FermiumASM/CensoredASM tweaks are enabled by default, but RLMixins aren't.

I'll download and test it, my guess is it is being caused by the singleton events config if you have that enabled.

**Comment by fonnymunkey** [comment]
(I can't actually download it, you have the file set as private requiring permission)

**Comment by XvaleX10** [comment]
have you found a solution?

**Comment by fonnymunkey** [comment]
No, your file is still undownloadable, its marked as a violation of Google Drive's TOS

**Comment by XvaleX10** [comment]
https://drive.google.com/file/d/1y5kcpvCPWNBe8WBrwfQ2jecUE0WIdRiv/view?usp=sharing
it should work now

**Comment by fonnymunkey** [comment]
I am not reproducing the exact same error regarding LittleTiles, however I am consistently reproducing the errors involving blocks such as BlockPhoenixEgg, FossilSaplingBlock, DrumBlock, etc. which are erroring for the same reasons as the LittleTiles errors.
(The blocks being created as falling blocks clientside and erroring because of the metadata not matching)

I am getting these same errors with the pack you sent as default, but also when running it with FermiumBooter, FermiumASM, and RLMixins disabled, which leads me to believe it is some other mod you have in your pack causing these errors.

Given its errors from these blocks being treated as falling blocks, maybe if you have some kind of physics mod, or something that modifies packets? It seems to be a situation of just removing a bunch of mods, and retesting until you narrow down which one is causing it.

**Comment by XvaleX10** [comment]
> I am not reproducing the exact same error regarding LittleTiles, however I am consistently reproducing the errors involving blocks such as BlockPhoenixEgg, FossilSaplingBlock, DrumBlock, etc. which are erroring for the same reasons as the LittleTiles errors. (The blocks being created as falling blocks clientside and erroring because of the metadata not matching)
> 
> I am getting these same errors with the pack you sent as default, but also when running it with FermiumBooter, FermiumASM, and RLMixins disabled, which leads me to believe it is some other mod you have in your pack causing these errors.
> 
> Given its errors from these blocks being treated as falling blocks, maybe if you have some kind of physics mod, or something that modifies packets? It seems to be a situation of just removing a bunch of mods, and retesting until you narrow down which one is causing it.

strange, yet when I tested it by removing FermiumASM it no longer happened

**Comment by fonnymunkey** [comment]
Here's the log for reference, all 3 disabled, I just created a new default world, went in spectator, and flew in one direction for about a minute: https://mclo.gs/e4ni3Rf

**Comment by fonnymunkey** [comment]
Closing as assumed unrelated due to errors still appearing without FermiumBooter/FermiumASM.
